romana/cmd/policy.go

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/romana/core/romana/util"
+
+	cli "github.com/spf13/cobra"
+	config "github.com/spf13/viper"
+)
+
+// policyCmd represents the policy commands
+var policyCmd = &cli.Command{
+	Use:   "policy [add|remove|list]",
+	Short: "Add, Remove or List a policy.",
+	Long: `Add, Remove or List a policy.
+
+For more information, please check http://romana.io
+`,
+}
+
+func init() {
+	policyCmd.AddCommand(policyAddCmd)
+	policyCmd.AddCommand(policyRemoveCmd)
+	policyCmd.AddCommand(policyListCmd)
+}
+
+var policyAddCmd = &cli.Command{
+	Use:          "add [tenantName][policyFile]",
+	Short:        "Add a new policy.",
+	Long:         `Add a new policy.`,
+	RunE:         policyAdd,
+	SilenceUsage: true,
+}
+
+var policyRemoveCmd = &cli.Command{
+	Use:          "remove [tenantName][policyName]",
+	Short:        "Remove a specific policy.",
+	Long:         `Remove a specific policy.`,
+	RunE:         policyRemove,
+	SilenceUsage: true,
+}
+
+var policyListCmd = &cli.Command{
+	Use:          "list [tenantName]",
+	Short:        "List policy for a specific tenant.",
+	Long:         `List policy for a specific tenant.`,
+	RunE:         policyList,
+	SilenceUsage: true,
+}
+
+// policyAdd adds kubernetes policy for a specific tenant
+// using the policyFile provided.
+func policyAdd(cmd *cli.Command, args []string) error {
```

Comment:
So this is kubernetes specific? It doesn't say that above... I am a bit confused.